application/clicommands/HealthCommand.php

```diff
@@ -37,9 +38,21 @@ public function checkAction()
             $health->setDbResourceName($name);
         }
 
+        if ($this->params->get('name')){
```

Comment:
This can be simplified to the following:
```php
$checkName = $this->params->get('name');
```

`Params::get()` returns `null` by default if the argument is not specified.

Please also note the change from `$check_name` to `$checkName`. We use camelCase instead of snake_case for class, method, property, variable names, etc.